src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -487,9 +496,9 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         return {};
     }
 
-    std::optional<til::point> ControlCore::GetHoveredCell() const
+    Windows::Foundation::IReference<Core::Point> ControlCore::HoveredCell() const

```

Comment:
depending on how often this is called, it could be quite expensive. an IReference is a heap allocation for every .. thing.